File: Perl/Avatars.pl
#!/usr/bin/perl
use strict;
use HTML::Template;
print qq(Content-type: text/html\n\n);
my @friends = ( 'notch', 'yive', 'jeb_' );
while (my ($key, $friend) = each @friends) {
	print '<img src="http://mcapi.ca/avatar/' . $friend . '"/>';
}

File: Python/Avatars.py
#! /usr/bin/python
# Thanks Sam from MinecraftMarket.com for helping me out with this language.
friends = [ 'yive', 'Notch', 'jeb_' ]
print "Content-type: text/html\n\n";
for friend in friends:
	print('<img src="http://mcapi.ca/avatar/%s"/>') % friend
